cpp/update_full_text_db.cc

```diff
@@ -327,12 +327,6 @@ bool ProcessRecordUrls(MARC::Record * const record, const unsigned pdf_extractio
         }
         if (not at_least_one_expired)
             return true;
-        else {
-            for (auto url_and_text_type(urls_and_text_types.begin()); url_and_text_type != urls_and_text_types.end(); ++url_and_text_type)
```

Comment:
Why not an implicit for-loop?

Reply:
This is left over debug code that has been removed (cf. 7a22ef8).